website/docs/reference/supported_commands.md

```diff
@@ -21,7 +21,7 @@ sidebar_position: 1
 | `find`            |                         | ✅     | Basic command is fully supported                          |
 |                   | `filter`                | ✅     |                                                           |
 |                   | `sort`                  | ✅     |                                                           |
-|                   | `projection`            | ✅     |                                                           |
+|                   | `projection`            | ✅     | Basic projection with field is supported                  |
```

Comment:
```suggestion
|                   | `projection`            | ✅     | Basic projections with fields is supported                  |
```

Probably? I'm not sure

Reply:
That sounds better than what I wrote, so lets take that :hugs: